passive_walker:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 3

  # Effort Controllers ---------------------------------------
  joint1_effort_controller:
    type: effort_controllers/JointEffortController
    joint: leg1_joint
    

  joint2_effort_controller:
    type: effort_controllers/JointEffortController
    joint: leg2_joint
    
    